pkg/services/ngalert/state/state_tracker.go

```diff
@@ -147,7 +147,14 @@ func (st *StateTracker) setNextState(alertRule *ngModels.AlertRule, result eval.
 			EvaluationState: result.State,
 		})
 		if currentState.State == eval.Alerting {
-			currentState.EndsAt = result.EvaluatedAt.Add(alertRule.For * time.Second)
+			//TODO: Move me and unify me with the top level constant
```

Comment:
There's a top-level constant in the `ngalert` file but I can't import that - I'll re-organise the eval / schedule package s and in a further refactor.

For now, I just master to be fixed.